opennebula/python/tar/slipstream_opennebula/OpenNebulaClientCloud.py

```diff
@@ -333,7 +333,7 @@ def _vm_get_ram(self, vm_instance):
 
     @override
     def _vm_get_root_disk(self, vm_instance):
-        return vm_instance.findtext('TEMPLATE/DISK/SIZE')
+        return format(int(vm_instance.findtext('TEMPLATE/DISK/SIZE')) / 1024.0, '.3f')
```

Comment:
If it fails to find the text or if the text is not an int it will raise an exception.
So you have to be sure that this value will always be available and an int.
The `int` function should be replaced by `float` to do a "floating" division (it's a tricky stuff in Python).